tests/dummy/app/templates/components/collapsible-group.hbs

```diff
@@ -1,6 +1,6 @@
 <li class="ember-power-select-group" aria-disabled={{ember-power-select-true-string-if-present disabled}} role="option">
   <span class="ember-power-select-group-name collapsible {{if collapsed 'is-collapsed'}}" onclick={{toggle 'collapsed' this}}>{{groupName}}</span>
-  {{#unless collapsed}}
+  <div class="{{if collapsed 'hide-options'}}">
```

Comment:
I almost bet money that it's because of `https://github.com/cibernox/ember-power-select/blob/master/addon/components/power-select.js#L562`